ftw/news/browser/news_listing_block.py

```diff
@@ -128,6 +129,10 @@ def get_item_dict(self, brain):
             image_tag = obj.restrictedTraverse('@@leadimage')(
                 'news_listing_image')
 
+        show_review_state = self.context.show_review_state and \
+            api.user.has_permission('ftw.contentpage: Add News', obj=aq_parent(obj))
```

Comment:
Looks like this should be ftw.news: Add News ?

Reply:
👍 

There is such a permission in ftw.contentpage. That the reason, why the tests haven't been failed with this permission.

I updated it with the proper permission.